m4_sessions/1710-inning/group_two.md

```diff
@@ -5,22 +5,43 @@ Session Title: Rule #6 - Don't take yourself so seriously...
 Description: Why should students come to this session? 
 Learn why it's important to take a step back. Get a better perspective and a more light hearted approach to...everything. 
 
-You're not that important. Laugh at the joke called "you"?
+You're not that important. Learn to laugh at the joke called "you".
 
 How does your mentality affect growth, perception and life in general.
 
 What's really important?
 
-
 Session Outcomes: What will students leave with or gain from this session?
 
 Perspective. Identifying where and how to let go. What's the balance between serious and too serious? (and candy).
 
 Session Outline:  Include the flow of the session and the time for each section.
 
 1 hour 15 minutes total
-10 minutes - introduction
-10 minutes - Group participation, review of questions
+5-10 minutes - Introduction
+2 minutes - video for quick intro - https://www.youtube.com/watch?v=M-HG6X6fpBM
+1 minute - restate what rule #6
+3-5 minute reflection:
+  - What are the negative impacts on what taking yourself too seriously can have?
+  - What does it mean to take yourself too seriously?
+  - We might play Manamana in the background during this (https://www.youtube.com/watch?v=8N_tupPBtWQ)
+10 - 20 minutes - Group discussion
+  - Ask someone to share (start tossing candy at people for answering)
```

Comment:
You might want to do a pair share before whole group (or break into smaller groups) just to get people started